foundation-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+### June 22, 2021
+
+#### iOS Core 2.9.3
```

Comment:
`### iOS Core 2.9.3`

Reply:
we may have done this for previous releases, that is use the wrong tag size

Reply:
I just compared it to the previous section not the older ones, yes, we should unify the heading format.